.gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+# Cache gems in between builds
```

Comment:
MailRoom uses travis ci: https://travis-ci.org/tpitale/mail_room.

Are there reasons why we would _also_ run gitlab ci?